src/components/TagInput/TagInput.styles.ts

```diff
@@ -0,0 +1,104 @@
+import { BaseProps } from '@/types';
+
+type InputConteinerConfig = BaseProps & {
+  isValidationError: BaseProps;
+};
+
+export type TagInputConfig = BaseProps & {
+  innerElements?: {
+    input?: BaseProps;
+    inputContainer?: InputConteinerConfig;
+    label?: BaseProps;
+    helperText?: BaseProps;
+  };
+};
+
+export const defaultConfig = {
+  display: 'flex',
+  flexDirection: 'column',
+  height: '40px',
+  gap: '$space-component-gap-small',
+  w: '320px',
+  padding:
+    '$space-component-padding-small $space-component-padding-large $space-component-padding-small $space-component-padding-small',
+  flexShrink: 0,
+  innerElements: {
+    inputContainer: {
+      w: '100%',
+      h: '$size-medium',
+      borderRadius: '$border-radius-large',
+      border: '1px solid $color-interaction-border-neutral-normal',
+      display: 'flex',
+      gap: '$space-component-gap-small',
+      alignItems: 'center',
+      bg: '$color-interaction-background-formField',
+      borderColor: 'rgb(207, 214, 221)',
+      p: '$space-component-padding-small $space-component-padding-large $space-component-padding-small $space-component-padding-small',
+      flexShrink: '0',
+      ringInset: 'true',
+      ring: '$border-width-small',
+      boxShadow: '0 0 #0000',
+      ringColor: {
+        _: '$color-interaction-border-neutral-normal',
+        hoverWithoutButton: '$color-interaction-border-hover',
+        alert: '$color-interaction-border-alert',
+        focusWithin: '$color-interaction-neutral-subtle-normal',
+      },
+      outlineStyle: {
+        '&:has(input:focus)': 'solid',
+      },
+      outlineColor: {
+        focusWithin: '$color-interaction-focus-default',
+      },
+      outlineWidth: {
+        '&:has(input:focus)': '$border-width-focus',
+      },
+      outlineOffset: {
+        _: '1px',
+        alert: '$border-width-focus',
+      },
+      transition: 'true',
+      transitionDuration: '50',
```

Comment:
I feel like transition-duration should be kept consistent, let me get back to you, which value to use it here

Reply:
I have no idea how to get information about it - `Tag` has `transitionDuration: '50'`. Let me know if you get to know sth more

Reply:
I think you can keep the 50ms there for now, please add TODO: unify transitionDuration there somewhere

Reply:
ok, done